src/services/projectsIssues/projectsIssuesTypes.ts

```diff
@@ -0,0 +1,44 @@
+export interface Error {
```

Comment:
Similarly, there's an ErrorData type in the general src/types folder. Maybe this and the rejection type can be moved there since they will probably also be used in other parts of the programme?